book/proposal/proposal.md

```diff
@@ -0,0 +1,99 @@
+# Proposal for Design Think for Developers
```

Comment:
It's possible the publisher won't want to run with "Design" in the title, as it's easy to mistake for "Visual Design." In any case, you'll almost certainly be asked to provide several titles & subtitles for the book as part of your early interaction with your development editor.

It might be good to prepare some options. (This was one of the more surprisingly difficult tasks for me—besides writing the marketing copy and picking out a cover).